File: app/src/main/java/com/bryan/platform/PlatformApplication.java
package com.bryan.platform;

import android.app.Application;
import android.content.Context;

public class PlatformApplication extends Application {

    private static PlatformApplication instance;

    public static Context applicationContext() {
        return instance.getApplicationContext();
    }

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;
    }
}

File: app/src/main/java/com/bryan/platform/PlatformApplication.kt
package com.bryan.platform

import android.app.Application
import android.content.Context

class PlatformApplication : Application() {

    companion object {
        private var instance: PlatformApplication? = null

        fun applicationContext(): Context {
            return instance!!.applicationContext
        }
    }

    override fun onCreate() {
        super.onCreate()
        instance = this
    }
}